tests/unit/lms/services/group_info_test.py

```diff
@@ -0,0 +1,124 @@
+from unittest import mock
+
+import pytest
+
+from lms.models import ApplicationInstance, GroupInfo
+from lms.services.group_info import GroupInfoService
+
+
+class TestGroupInfoUpsert:
+    AUTHORITY = "TEST_AUTHORITY_PROVIDED_ID"
+    CONSUMER_KEY = "TEST_CONSUMER_KEY"
```

Comment:
I'm not sure these add much beyond just using literal `"TEST_AUTHORITY_PROVIDED_ID"` wherever needed.

`AUTHORITY` should at least be renamed since it's not an authority (a word that has a specific meaning in Hypothesis) but an authority_provided_id.

Reply:
The intention is to: 

 * Prevent typos
 * Indicate to the reader that this is _the_ value, rather than just _a_ value when compared with
 * Generally it's a good idea in classes not to have repeated literal values. As in code, so in tests etc.

Reply:
Alright, I get can get on board with this one. As as a general pattern, maybe all test classes should start declaring their CONSTANTS as class attrs.

I do wonder about putting these at the bottom of the class though (maybe above the class's fixtures and helper methods) instead of at the top. I know the top of a module or class is the normal place to put a constant. But we usually put all our test fixtures and helper methods at the bottom of the class just to get them out of the way, so you don't have to scroll past a bunch of noise and boilerplate before getting to the tests. After all I shouldn't usually need to know what the string value of `CONSUMER_KEY` is.

```diff
diff --git a/tests/unit/lms/services/group_info_test.py b/tests/unit/lms/services/group_info_test.py
index 8a944f08..b2194b09 100644
--- a/tests/unit/lms/services/group_info_test.py
+++ b/tests/unit/lms/services/group_info_test.py
@@ -7,9 +7,6 @@ from lms.services.group_info import GroupInfoService
 
 
 class TestGroupInfoUpsert:
-    AUTHORITY = "TEST_AUTHORITY_PROVIDED_ID"
-    CONSUMER_KEY = "TEST_CONSUMER_KEY"
-
     def test_it_adds_a_new_GroupInfo_if_none_exists(
         self, db_session, group_info_svc, params
     ):
@@ -75,6 +72,9 @@ class TestGroupInfoUpsert:
             .one()
         )
 
+    AUTHORITY = "TEST_AUTHORITY_PROVIDED_ID"
+    CONSUMER_KEY = "TEST_CONSUMER_KEY"
+
     @pytest.fixture
     def group_info_svc(self, pyramid_config, pyramid_request):
         """The group_info_svc service that is being tested."""
```